libs/island-ui/core/src/lib/Footer/Footer.tsx

```diff
@@ -31,6 +31,7 @@ interface FooterProps {
   hideLanguageSwith?: boolean
   showMiddleLinks?: boolean
   showTagLinks?: boolean
+  showDrawerMenu?: boolean
```

Comment:
would this be more descriptive if it was something like "hasDrawerMenu" ? I get the feeling like this is showing or hiding a drawer menu

Reply:
yes! makes sense